db/config.cc

```diff
@@ -849,6 +849,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , override_decommission(this, "override_decommission", value_status::Used, false, "Set true to force a decommissioned node to join the cluster (cannot be set if consistent-cluster-management is enabled")
     , enable_repair_based_node_ops(this, "enable_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, true, "Set true to use enable repair based node operations instead of streaming based")
     , allowed_repair_based_node_ops(this, "allowed_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, "replace,removenode,rebuild,bootstrap,decommission", "A comma separated list of node operations which are allowed to enable repair based node operations. The operations can be bootstrap, replace, removenode, decommission and rebuild")
+    , enable_compacting_data_for_streaming_and_repair(this, "enable_compacting_data_for_streaming_and_repair", liveness::LiveUpdate, value_status::Used, true, "Enable compacting the data for streaming and repair (load'n'stream included). Can reduce the amount of data to be processed, but adds CPU overhead.")
```

Comment:
We can make it clear this is option is not about the normal compaction or off-startegy compaction after repair or streaming.

How about:

Enable the compacting reader which compacts data before sending or synchronizing  data to the peer node. This allows skipping sending data covered by tombstones and reducing the amount of data to be processed, but adds CPU overhead.

Reply:
Done in v4.